server/app.js

```diff
@@ -39,6 +40,7 @@ app.use("/products", productsRouter);
 app.use("/upload-image", imageUploadRouter);
 app.use("/followers", followRouter);
 app.use("/notification", notificationRouter);
+app.use("/publicLists", publicListsRouter);
```

Comment:
routes should be hyphen-spaced. (don't forget to changes references in both server-side and client-side)